src/main/java/analyzer/exercises/needforspeed/AvoidConditionalLogic.java

```diff
@@ -0,0 +1,15 @@
+package analyzer.exercises.needforspeed;
+
+import analyzer.Comment;
+
+public class AvoidConditionalLogic extends Comment {
```

Comment:
Can you make the comments here package-private? That makes it a little harder for other exercise analyzers to accidentally reuse them.

Reply:
Done!